catchpoint-mixin/targets.libsonnet

```diff
@@ -9,4 +9,99 @@ local utils = commonlib.utils {
   new(this): {
     local vars = this.grafana.variables,
     local config = this.config,
+    local testNameLabel = config.testNameLabel,
+    local nodeNameLabel = config.nodeNameLabel,
+    local testAndNodeLabel = config.nodeNameLabel + config.testNameLabel,
+
+    topAvgLoadTimeTestName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (test_name) (avg_over_time(catchpoint_load_time{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topAvgTotalLoadTimeNodeName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (node_name) (avg_over_time(catchpoint_load_time{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(nodeNameLabel)),
+
+    topAvgDocumentCompletionTimeTestName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (test_name) (avg_over_time(catchpoint_document_complete_time{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topAvgDocumentCompletionTimeNodeName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (node_name) (avg_over_time(catchpoint_document_complete_time{%(pureTestNameSelector)s}[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(nodeNameLabel)),
+
+    bottomAvgRequestSuccessRatioTestName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'bottomk(1, avg by (test_name) (avg_over_time(((catchpoint_requests_count{%(pureTestNameSelector)s} - catchpoint_failed_requests_count{%(pureTestNameSelector)s}) / clamp_min(catchpoint_requests_count{%(pureTestNameSelector)s},1))[$__interval:])))' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s - success' % utils.labelsToPanelLegend(testNameLabel)),
+
+    topAvgFailedRequestRatioNodeName:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk(1, avg by (node_name) (avg_over_time(((catchpoint_failed_requests_count{%(pureTestNameSelector)s}) / clamp_min(catchpoint_requests_count{%(pureTestNameSelector)s},1))[$__interval:])))' % vars
```

Comment:
I'd change this query to better reflect the  `bottomAvgRequestSuccessRatioTestName` but with Node Name instead.